clash-prelude/src/Clash/Sized/Vector.hs

```diff
@@ -33,7 +33,7 @@ module Clash.Sized.Vector
     -- ** Length information
   , length, lengthS
     -- ** Indexing
-  , (!!), head, last, at
+  , (!!), index_int, head, last, at
```

Comment:
How come this export is needed, but the one for `replace_int` is not?

Reply:
Ah right, I had intentionally exported them, but it caused test failures (due to a test defining `replace_int`). I then changed my mind and just used a string again.. I'll do that for `index_int` too. I think the real(tm) solution is to create an `Internal` module.